src/commands/outputMetrics.ts

```diff
@@ -42,10 +57,16 @@ export function outputAsJson(
 
 function buildOutputObject(
     fileMetrics: Map<string, Map<string, MetricResult>>,
+    unknownFiles: string[],
     relationshipMetrics: CouplingResult
 ) {
-    const output: { nodes: OutputNode[]; relationships: OutputRelationship[] } = {
+    const output: {
+        nodes: OutputNode[];
+        info: OutputInfoNode[];
```

Comment:
replace "info" by "unknownFile" would make more sense to me

Reply:
clarified